yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -168,6 +166,9 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 				return err
 			}
 		}
+		(*conn).Close(context.Background())
```

Comment:
Ah I see. when there is an error, we want to close the connection anyway, whether we report it or not, correct? 

nit: How about we write the following on line 156 (as soon as we enter the err != nil block). That way, you won't have to have the close connection logic in two places (one here, and one on line 87)
```
defer func() {
    (*conn).Close(context.Background())
    *conn = nil
}
```

Reply:
or if you can close it there itself, no need of a defer.